pkgs/by-name/go/google-chrome/package.nix

```
@@ -159,116 +168,202 @@ let
       gtk4
     ];
 
-in
-stdenv.mkDerivation (finalAttrs: {
-  pname = "google-chrome";
-  version = "127.0.6533.88";
+  linux = stdenv.mkDerivation (finalAttrs: {
+    inherit pname meta;
+    version = "127.0.6533.88";
 
-  src = fetchurl {
-    url = "https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_${finalAttrs.version}-1_amd64.deb";
-    hash = "sha256-0l9cidNFO0dcyzWy4nDD/OGFQDBLXx9aPVq6ioDkqK0=";
-  };
+    src = fetchurl {
+      url = "https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_${finalAttrs.version}-1_amd64.deb";
+      hash = "sha256-0l9cidNFO0dcyzWy4nDD/OGFQDBLXx9aPVq6ioDkqK0=";
+    };
+
+    strictDeps = true;
+
+    nativeBuildInputs = [
+      makeWrapper
+      patchelf
+    ];
+
+    buildInputs = [
+      adwaita-icon-theme
+      glib
+      gsettings-desktop-schemas
+      gtk3
+    ];
```

```suggestion
  buildInputs = [
    # needed for XDG_ICON_DIRS
    adwaita-icon-theme
    glib
    gtk3
    gtk4
    # needed for GSETTINGS_SCHEMAS_PATH
    gsettings-desktop-schemas
  ];
```

Please keep it the same as they were!